app/role_data.go

```diff
@@ -298,6 +301,12 @@ func addRoles(management *config.ManagementContext) (string, error) {
 		addRule().apiGroups("monitoring.cattle.io").resources("prometheus").verbs("view").
 		setRoleTemplateNames("view")
 
+	rb.addRoleTemplate("Manage Istio Networking", "istionetwork-manage", "project", true, false, false, false).
```

Comment:
Do we really need to build these in? From a practical perspective, who would ever use them?